packages/accordion/src/index.js

```diff
@@ -0,0 +1,406 @@
+////////////////////////////////////////////////////////////////////////////////
+// Welcome to @reach/accordion!
+
+import React, {
+  Children,
+  cloneElement,
+  createContext,
+  forwardRef,
+  useCallback,
+  useContext,
+  useMemo,
+  useRef,
+  useState
+} from "react";
+import { makeId, wrapEvent, useForkedRef } from "@reach/utils";
+import { useId } from "@reach/auto-id";
+import PropTypes from "prop-types";
+import warning from "warning";
+
+// A11y reference:
+//   - https://www.w3.org/TR/wai-aria-practices/examples/accordion/accordion.html
+//   - https://inclusive-components.design/collapsible-sections/
+
+// TODO: Screen reader testing
+
+const AccordionContext = createContext({});
+const AccordionItemContext = createContext({});
+const useAccordionContext = () => useContext(AccordionContext);
+const useAccordionItemContext = () => useContext(AccordionItemContext);
+
+////////////////////////////////////////////////////////////////////////////////
+// Accordion
+
+export const Accordion = forwardRef(function Accordion(
+  {
+    children,
+    defaultIndex,
+    index: controlledIndex,
+    onChange,
+    readOnly = false,
+    toggle = false,
```

Comment:
I don't understand this prop. What does it do? Is it like, "can completely collapse"?

Reply:
Correct. See `data-allow-toggle` in https://www.w3.org/TR/wai-aria-practices/examples/accordion/accordion.html